servicetalk-transport-netty-internal/src/test/java/io/servicetalk/transport/netty/internal/RequestResponseCloseHandlerTest.java

```diff
@@ -175,6 +178,7 @@ public Scenarios(final Mode mode, final List<Events> events, final ExpectEvent e
             SR,         // validate Socket TCP RST -> SO_LINGER=0
             UC,         // emit User Closing
             IH, OH, FC, // validate Input/Output Half-close, Full-Close
+            CE,         // emit ChannelOutputClosingEvent
```

Comment:
Till now these events represented things that happened on the socket or what is exposed on the Close handler APIs. `ChannelOutputClosingEvent` is an event generated by the `CloseHandler` itself. Can we avoid expanding scope of events here? Instead validate/simulate things that will happen on the socket, like aborting writes.